engine/events/map_name_sign.asm

```diff
@@ -276,7 +280,18 @@ LoadMapNameSignGFX:
 	rst AddNTimes ; preserves bc
 	ld de, wBGPals1 palette PAL_BG_TEXT
 	call FarCopyColorWRAM
-	jmp SetDefaultBGPAndOBP
+
+	; copy sign palette for PAL_BG_TEXT
+	ld hl, SignPals
+	ld bc, 1 palettes
+	ld a, [wSign]
+	rst AddNTimes ; preserves bc
+	ld de, wBGPals2 palette PAL_BG_TEXT
+	call FarCopyColorWRAM
```

Comment:
This is mostly a duplicate of the paragraph right above it. The upshot is to copy the sign pal into `wBGPals2` as well as `wBGPals1`, so this could reuse the `hl` value computed above.
```suggestion
	; copy sign palette for PAL_BG_TEXT
	ld hl, SignPals
	ld bc, 1 palettes
	ld a, [wSign]
	rst AddNTimes ; preserves bc
	ld de, wBGPals2 palette PAL_BG_TEXT
	call FarCopyColorWRAM
```